.github/workflows/build_webrtc.yml

```diff
@@ -30,15 +35,19 @@ jobs:
     - name: Add depot tools to path
       run: |
         echo "$(pwd)/depot_tools" >> $GITHUB_PATH
-    
+
     - name: Fetch WebRTC revision
       run: |
         python3 build.py fetch --revision branch-heads/${{  inputs.branch_number }}
 
     - name: Build WebRTC
       run: |
         python3 build.py build
-        mv libwebrtc.aar libwebrtc-${{ inputs.milestone }}.aar
+      
+    - name: Get and rename built files
+      run: |
+        cp ./src/libwebrtc.aar libwebrtc-${{ inputs.milestone }}.aar
```

Comment:
moving instead of copying would take less space right?

Reply:
Oh yep mb